Tests/.swiftlint.yml

```diff
@@ -23,3 +23,7 @@ disabled_rules:
   # for tests. 
   - type_body_length
   - file_length 
+  # It's recommended to not use force casts in test suite because if a cast fails, the test suite will immediately stop executing. 
```

Comment:
Multiple times now, I have gone into this file to disable `force_cast` rule, thinking it isn't a big deal to allow this rule in our test classes. But, then I remember that there are consequences to it. 

So, adding this comment block to remind others of the consequences to disabling force_cast lint rule.